go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/equinix/terraform-provider-equinix
 
-go 1.22
+go 1.22.0
```

Comment:
We don't specify patch versions in other Go things (whether shipped as modules or binaries).  What makes it necessary here?

Reply:
When tools try to fetch the toolchain for this project they find `1.22`, which is not a release version -- there is no toolchain defined without full semver. 

```sh
$ go version
go: downloading go1.22 (linux/amd64)
go: download go1.22 for linux/amd64: toolchain not available
```

* https://github.com/golang/go/issues/62278#issuecomment-1693538776
* https://go.dev/doc/toolchain

An alternative approach is to specify `toolchain 1.22.0`.

Reply:
Does that error happen for you in `metal-cli` and `equinix-sdk-go` as well, or only here?  And it's preventing you from running `go mod tidy` or you're trying to do something else?  That doesn't seem to be impacting me or Renovate, but I'm not clear exactly how to reproduce this problem.